helm/examples/azure/sa/create-secret.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+WORK_DIR=$(dirname $0)
+source ${WORK_DIR}/env-sa
+
+[[ -n "$SUBSCRIPTION" ]] && az account set --subscription $SUBSCRIPTION
+
+VALUES_FILE=${WORK_DIR}/../hpcc-azurefile/values.yaml
+SECRET_NAME_INPUT=$(cat $VALUES_FILE | grep "^[[:space:]]*secretName:"|cut -d':' -f2|sed 's/[[:space:]\"]//g')
+SECRET_NAMESPACE_INPUT=$(cat $VALUES_FILE | grep "^[[:space:]]*secretNamespace:"|cut -d':' -f2|sed 's/[[:space:]\"]//g')
+[[ -n "$SECRET_NAME_INPUT" ]] && SECRET_NAME=${SECRET_NAME_INPUT}
+[[ -n "$SECRET_NAMESPACE_INPUT" ]] && SECRET_NAMESPACE=${SECRET_NAMESPACE_INPUT}
+
+
+if [[ -z "$SECRET_NAME" ]] || [[ -z "$SECRET_NAMESPACE" ]]
+then
+  echo "Miss one of SECRET_NAME and SECRET_NAMESPACE"                                                                           exit 1
```

Comment:
'exit 1' should be on the next line; it is currently spaced far to the right of the echo line